src/initramfs/cmd/init/pkg/mount/mount.go

```diff
@@ -334,12 +336,22 @@ func probeDevice(devname string) (*BlockDevice, error) {
 	if err != nil {
 		return nil, err
 	}
+	PARTLABEL, err := blkid.ProbeLookupValue(pr, "PARTLABEL", nil)
```

Comment:
I wonder if we should switch over to `PARTLABEL` to lookup the `ROOT` and `DATA` partitions. If that is the convention used elsewhere. Doesn't need to be included here, good followup ticket.